commons/impl/src/i18n/org/sakaiproject/commons/impl/ui.properties

```diff
@@ -43,4 +43,4 @@ link_to = Link to:
 load_thumbnail = Load thumbnail:
 link_button_tooltip = Insert a link
 image_button_tooltip = Insert an image
-choose_file = Choose file:
+choose_image_file = Choose an image file. It must be a png, gif or jpeg:
```

Comment:
please use oxford comma

Reply:
Are you taking the piss? The oxford comma does not apply here. The meaning is obvious.

Reply:
It's software consistency ..... we use oxford comma everywhere else. why would your new text be different?

Reply:
I thought you were having a laugh. It's been a long day.

Reply:
I think being funny would have requested use of the Cambridge or Lancaster comma

Reply:
We need a Sorbonne comma, too, or just a general 'Old Europe comma' for all the euro translators to use :)